src/appsco-manage-application-page.html

```diff
@@ -164,6 +164,10 @@
                         preview></appsco-application-assignees>
 
                 <div class="info">
+                    Created by: [[ application.owner.email ]]
```

Comment:
better is maybe to use display_name rather then email. If first name is not present email will be default